docs/events/init.md

```diff
@@ -0,0 +1,58 @@
+# INIT
+
+**async | master**
+
+The `INIT` event is triggered before the `run` or `readTests tasks are executed. The event handler can be asynchronous: in this case, the tasks will start executing only after the _Promise_ returned by the event handler is resolved. The event is triggered only once, regardless of how many times the tasks are executed.
+
+## Subscription {#subscription}
+
+```javascript
+testplane.on(testplane.events.INIT, async () => {
+    console.info('Processing INIT event…');
+});
+```
+
+### Handler parameters {#handler-parameters}
+
+No data is passed to the event handler.
+
+## Example of using {#usage}
+
+In the `INIT` event handler, you can organize, for example, the launch of a dev server for your project.
```

Comment:
hmm as far as i remember we now have a dedicated API for launching a dev server. Advising here to do it in the INIT handler is a bit misleading IMO.